test/sra_test.ts

```diff
@@ -1,73 +1,370 @@
-import { expect } from '@0x/contracts-test-utils';
+import { ContractAddresses, getContractAddressesForChainOrThrow } from '@0x/contract-addresses';
+import { DummyERC20TokenContract, WETH9Contract } from '@0x/contracts-erc20';
+import { constants, expect, OrderFactory } from '@0x/contracts-test-utils';
+import { BlockchainLifecycle, web3Factory, Web3ProviderEngine } from '@0x/dev-utils';
+import { assetDataUtils, Order, orderHashUtils } from '@0x/order-utils';
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
 import * as HttpStatus from 'http-status-codes';
 import 'mocha';
 
 import * as config from '../src/config';
-import { DEFAULT_PAGE, DEFAULT_PER_PAGE, SRA_PATH } from '../src/constants';
+import { DEFAULT_PAGE, DEFAULT_PER_PAGE, NULL_ADDRESS, SRA_PATH } from '../src/constants';
 import { getDBConnectionAsync } from '../src/db_connection';
-import { SignedOrderEntity } from '../src/entities';
+import { ErrorBody, GeneralErrorCodes, generalErrorCodeToReason, ValidationErrorCodes } from '../src/errors';
+import { APIOrderWithMetaData } from '../src/types';
+import { orderUtils } from '../src/utils/order_utils';
 
-import * as orderFixture from './fixtures/order.json';
-import { setupApiAsync, teardownApiAsync } from './utils/deployment';
-import { httpGetAsync } from './utils/http_utils';
+import { setupApiAsync, setupMeshAsync, teardownApiAsync, teardownMeshAsync } from './utils/deployment';
+import { constructRoute, httpGetAsync, httpPostAsync } from './utils/http_utils';
+import { DEFAULT_MAKER_ASSET_AMOUNT, MeshTestUtils } from './utils/mesh_test_utils';
 
-const SUITE_NAME = 'sra tests';
+const SUITE_NAME = 'Standard Relayer API (SRA) tests';
 
+const EMPTY_PAGINATED_RESPONSE = {
+    perPage: DEFAULT_PER_PAGE,
+    page: DEFAULT_PAGE,
+    total: 0,
+    records: [],
+};
+
+const TOMORROW = new BigNumber(Date.now() + 24 * 3600); // tslint:disable-line:custom-no-magic-numbers
+async function addNewSignedOrderAsync(
+    orderFactory: OrderFactory,
+    params: Partial<Order>,
+    remainingFillableAssetAmount?: BigNumber,
+): Promise<APIOrderWithMetaData> {
+    const order = await orderFactory.newSignedOrderAsync({
+        expirationTimeSeconds: TOMORROW,
+        ...params,
+    });
+    const apiOrder: APIOrderWithMetaData = {
+        order,
+        metaData: {
+            orderHash: orderHashUtils.getOrderHash(order),
+            remainingFillableTakerAssetAmount: remainingFillableAssetAmount || order.takerAssetAmount,
+        },
+    };
+    await (await getDBConnectionAsync()).manager.save(orderUtils.serializeOrder(apiOrder));
+    return apiOrder;
+}
 describe(SUITE_NAME, () => {
+    let chainId: number;
+    let contractAddresses: ContractAddresses;
```

Comment:
I removed some of the ceremony here by just making these constants (in constants.ts) in my PR.